src/features/profile/ProfilePage.tsx

```diff
@@ -47,7 +49,9 @@ const ProfilePage = ({ hasBerthNotifications, hasWSNotifications }: ProfilePageP
                 </span>
               </Tab>
             </TabList>
-            <TabPanel>Contact information tab</TabPanel>
+            <TabPanel>
+              <ContactInfo {...customerContactInfo} />
+            </TabPanel>
             <TabPanel>Boats tab</TabPanel>
             <TabPanel>Berths tab</TabPanel>
             <TabPanel>Winter storage tab</TabPanel>
```

Comment:
Not our components, but I wonder if it would be possible to pair the tabs and tab panels somehow. The implicit nature of "first tab points to first tab panel" is a bit prone to errors

Reply:
This is an excellent point, and I totally agree. 
I double-checked their docs and the source code, it looks like this is the only way to link them with each other.

https://github.com/City-of-Helsinki/helsinki-design-system/blob/4cee25c0f94a9b91e8fd5c763dd6fe7bcf7afb3a/packages/react/src/components/tabs/Tabs.tsx#L39